mobile-app/app/screens/TransactionAuthorization/TransactionAuthorization.tsx

```diff
@@ -194,19 +203,22 @@ export function TransactionAuthorization (): JSX.Element | null {
         })
         .catch(e => Logging.error(e)) // not expect logic reach here
         .finally(() => {
-          dispatch(transactionQueue.actions.pop()) // remove job
-          onTaskCompletion()
+          setTimeout(() => {
+            dispatch(transactionQueue.actions.pop()) // remove job
+            onTaskCompletion()
+          }, 2000)
         })
     } else if (authentication !== undefined) {
       emitEvent('BLOCK') // prevent any re-render trigger (between IDLE and PIN)
 
       setMessage(authentication.message)
-      setLoadingMessage(authentication.loading)
+      setLoadingMessage('Verifying access')
```

Comment:
```suggestion
      setLoadingMessage( authentication.loading ?? 'Verifying access')
```
Allow the caller to overwrite as the interface designated so. Remove from the caller side if want to change the end result.

Reply:
I agree, actually I'll just change what the caller is passing to keep ```setLoadingMessage(authentication.loading)```. Updated!